[2024-06-12T03:53:04.970+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: example_dag_with_callback.task5 manual__2024-06-12T03:50:37.880774+00:00 [queued]>
[2024-06-12T03:53:04.975+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: example_dag_with_callback.task5 manual__2024-06-12T03:50:37.880774+00:00 [queued]>
[2024-06-12T03:53:04.975+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-06-12T03:53:04.975+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-06-12T03:53:04.975+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-06-12T03:53:04.985+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): task5> on 2024-06-12 03:50:37.880774+00:00
[2024-06-12T03:53:04.989+0000] {standard_task_runner.py:55} INFO - Started process 1528 to run task
[2024-06-12T03:53:04.991+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dag_with_callback', 'task5', 'manual__2024-06-12T03:50:37.880774+00:00', '--job-id', '334', '--raw', '--subdir', 'DAGS_FOLDER/example_dag_with_callback.py', '--cfg-path', '/tmp/tmpm3lwzep1']
[2024-06-12T03:53:04.991+0000] {standard_task_runner.py:83} INFO - Job 334: Subtask task5
[2024-06-12T03:53:05.027+0000] {task_command.py:376} INFO - Running <TaskInstance: example_dag_with_callback.task5 manual__2024-06-12T03:50:37.880774+00:00 [running]> on host 3eba8accb13f
[2024-06-12T03:53:05.062+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_dag_with_callback
AIRFLOW_CTX_TASK_ID=task5
AIRFLOW_CTX_EXECUTION_DATE=2024-06-12T03:50:37.880774+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-06-12T03:50:37.880774+00:00
[2024-06-12T03:53:05.063+0000] {logging_mixin.py:120} INFO - Task 5 is completed
[2024-06-12T03:53:05.063+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-06-12T03:53:05.068+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=example_dag_with_callback, task_id=task5, execution_date=20240612T035037, start_date=20240612T035304, end_date=20240612T035305
[2024-06-12T03:53:05.083+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-06-12T03:53:05.093+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
